bcrs-ui/src/router/routes.ts

```diff
@@ -0,0 +1,25 @@
+import Home from '@/views/Home.vue'
+import MixinExample from '@/views/MixinExample.vue'
+import StateExample from '@/views/StateExample.vue'
+
+export const routes = [
+  {
+    path: '/',
+    name: 'home',
+    component: Home
+  },
+  {
+    path: '/MixinExample',
+    name: 'mixinExample',
+    component: MixinExample
+  },
+  {
+    path: '/StateExample',
+    name: 'stateExample',
+    component: StateExample
+  },
+  {
+    // default/fallback route
+    path: '*',
+    redirect: '/'
+  }]
```

Comment:
These should be on separate lines or may cause lint issue.

Reply:
Not an error in the frameworks linting or coops-ui, but will still separate for consistency 👍